### DISCUSSION TITLE
Dynamic resources

### DISCUSSION THREAD

**Discussion by Pennywise96** [filamentphp/filament#11157] [Help]
### Package

Panel builder

### Package Version

v.3.0.0

### How can we help you?

Hi!
I wanted to ask if there is a way to register dynamic resources. My goal is to create for example a DatasetResource and this resource can exist multiple times in a panel. So a user could create a new Dataset with a form and the resource would be created dynamically by the database fields the user has created.
The created resources whould share the table and form configuration which are set in the DatasetResource class.

I hope you have understood my question, thank you in advance!

**Comment by leandrocfe** [comment]
Something like this? (I've never tried this plugin before)
https://filamentphp.com/plugins/miguilim-auto-panel

**Reply by Pennywise96** [reply]
No, unfortunately that's not what I need... but thanks anyway!

**Comment by demianottema** [comment]
Did you find a suitable solution?
